apps/shipments/views.py

```diff
@@ -142,17 +143,18 @@ class LocationViewSet(viewsets.ModelViewSet):
     queryset = Location.objects.all()
     serializer_class = LocationSerializer
     # TODO: Clarify/Solidify the permissions for Locations w/ respect to owner_id
-    permission_classes = (permissions.IsAuthenticated, IsOwner) if settings.PROFILES_URL else (permissions.AllowAny,)
+    permission_classes = (permissions.IsAuthenticated, IsOwner) if settings.PROFILES_URL != 'DISABLED'\
```

Comment:
Wrap multi-line boolean statements in parenthesis so you can span them on multiple lines without adding line breaks ('\')